src/components/Konnector.jsx

```diff
@@ -1,17 +1,35 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import flow from 'lodash/flow'
 
 import { Routes as HarvestRoutes } from 'cozy-harvest-lib'
 import datacardOptions from 'cozy-harvest-lib/dist/datacards/datacardOptions'
+import { useWebviewIntent } from 'cozy-intent'
 
 import { getKonnector } from 'ducks/konnectors'
 
 import { getTriggersByKonnector } from 'reducers'
 
 const Konnector = ({ konnector, history, triggers }) => {
   const konnectorWithTriggers = { ...konnector, triggers: { data: triggers } }
+  const webviewIntent = useWebviewIntent()
+
+  useEffect(() => {
+    if (!webviewIntent) return
+
+    webviewIntent.call('setFlagshipUI', {
+      statusBarIcons: 'dark',
+      navbarIcons: 'dark'
+    })
+
+    return () =>
+      webviewIntent.call('setFlagshipUI', {
+        statusBarIcons: 'light',
+        navbarIcons: 'light'
+      })
+  }, [webviewIntent])
```

Comment:
nit: let's test:
- when webviewIntent null
- when component is mount
- when component is unmount

Reply:
Well, this type of webview hook will be seen in a lot of places (I'm using pretty much the same one in cozy-ui for instance). How should we test something that will be implemented the same everywhere while still maintaining meaningful tests? Maybe creating an extra hook that will handle all these hooks, I'm not sure? What would you say

Reply:
Simple unit testing is a solution, in my opinion